codec/src/main/java/io/netty/handler/codec/compression/ByteBufChecksum.java

```diff
@@ -33,14 +33,13 @@
  * byte array ({@link ByteBuf#hasArray()} is {@code true}) or not.
  */
 abstract class ByteBufChecksum implements Checksum {
-    private static final Method ADLER32_UPDATE_METHOD;
-    private static final Method CRC32_UPDATE_METHOD;
-
-    static {
-        // See if we can use fast-path when using ByteBuf that is not heap based as Adler32 and CRC32 added support
-        // for update(ByteBuffer) in JDK8.
```

Comment:
can we just duplicate the comment for both static fields so we dont loose why we did this ?

Reply:
yep good idea :+1: